Cli-CredentialHelper/Installer.cs

```diff
@@ -13,7 +13,6 @@ internal class Installer
         internal const string ParamPathKey = "--path";
         internal const string ParamPassiveKey = "--passive";
         internal const string ParamForceKey = "--force";
-        private static readonly Version NetFxMinVersion = new Version(4, 5, 1);
```

Comment:
I'd like to keep this, if for nothing else than a kind reminder of what we're stuck with. :wink: